pkg/cmd/resources/hpa.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if eq .Values.autoscaler "hpa" }}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "app.fullname" . }}-autoscaler
+  {{- with .Values.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "app.fullname" . }}
+  minReplicas: {{ .Values.replicas }}
+  maxReplicas: {{ .Values.maxReplicas }}
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        type: Utilization
+        averageUtilization: {{ .Values.targetCpuUtilizationPercentage }}
+  - type: Resource
+    resource:
+     name: memory
```

Comment:
This line needs to be indented one more space. I just discovered this bug and filed an issue: https://github.com/spinkube/spin-plugin-kube/issues/68